hw 2/templates/board.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Departure board</title>
+    <style>
+            table {
+                border-collapse: collapse;
+            }
+            th, td {
+                width:150px;
+                text-align:center;
+                border:1px solid black;
+                padding:5px
+    </style>
```

Comment:
You forgot to put closing curly bracket at the end of the scope!

Reply:
fixed issue